cypress/integration/pages/articles/index.js

```diff
@@ -1,6 +1,25 @@
 import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import tests from './tests';
-import testsForAMPOnly from './testsForAMPOnly';
-import testsForCanonicalOnly from './testsForCanonicalOnly';
+import { testsToAlwaysRun, tests, testsToNeverSmokeTest } from './tests';
+import {
+  testsToAlwaysRunForAMPOnly,
+  testsForAMPOnly,
+  testsToNeverSmokeTestForAMPOnly,
+} from './testsForAMPOnly';
+import {
+  testsToAlwaysRunForCanonicalOnly,
+  testsForCanonicalOnly,
+  testsToNeverSmokeTestForCanonicalOnly,
+} from './testsForCanonicalOnly';
 
-runTestsForPage('articles', tests, testsForCanonicalOnly, testsForAMPOnly);
+runTestsForPage(
```

Comment:
Should we consider passing in an object instead now there are so many args? If arg order is incorrect here it would be hard to spot.

```js
runTestsForPage({
  pageType: 'articles',
  testsToAlwaysRun,
  tests,
  testsToNeverSmokeTest,
  testsToAlwaysRunForCanonicalOnly,
  testsForCanonicalOnly,
  testsToNeverSmokeTestForCanonicalOnly,
  testsToAlwaysRunForAMPOnly,
  testsForAMPOnly,
  testsToNeverSmokeTestForAMPOnly,
})
```

Reply:
Working on this and discussing it now, problem is readability concerns after doing this.

Reply:
Done